lite2/client.go

```diff
@@ -757,6 +782,65 @@ func (c *Client) fetchHeaderAndValsAtHeight(height int64) (*types.SignedHeader,
 	return h, vals, nil
 }
 
+// fetchMissingTrustedHeader finds the closest height after the
+// requested height and does backwards verification.
+func (c *Client) fetchMissingTrustedHeader(height int64, now time.Time) (*types.SignedHeader, error) {
+	c.logger.Info("Fetching missing header", "height", height)
+
+	trustedHeader, err := c.trustedStore.SignedHeaderAfter(height)
+	if err != nil {
+		return nil, errors.Wrapf(err, "can't get signed header after %d", height)
```

Comment:
Suggestion: Instead of throwing back an error could we just use c.trustedHeader i.e. the latest trusted header of the lite client and use back verification from there.

Reply:
Don't think so. c.trustedHeader should also exist in the database. If it's not there, it means something is horribly wrong